packages/vuetify/src/components/VTreeview/__tests__/VTreeview.spec.ts

```diff
@@ -749,4 +749,41 @@ describe('VTreeView.ts', () => { // eslint-disable-line max-statements
 
     expect(input).not.toHaveBeenCalled()
   })
+
+  // https://github.com/vuetifyjs/vuetify/issues/10990
+  // https://github.com/vuetifyjs/vuetify/issues/10770
+  it('should not disable children of disabled parent when in independent mode', async () => {
```

Comment:
this passes regardless of the fix.

Reply:
that's partly because we didn't really check disabled correctly before for child nodes